services/catalog/src/simcore_service_catalog/api/dependencies/services.py

```diff
@@ -1,13 +1,13 @@
 from fastapi.requests import Request
-from models_library.services_resources import ServiceResources
+from models_library.services_resources import ResourcesDict
 
 from ...core.settings import AppSettings
 from ...models.schemas.services_specifications import ServiceSpecifications
 
 
-def get_default_service_resources(request: Request) -> ServiceResources:
+def get_default_service_resources(request: Request) -> ResourcesDict:
     app_settings: AppSettings = request.app.state.settings
-    return app_settings.CATALOG_SERVICES_DEFAULT_RESOURCE
+    return app_settings.CATALOG_SERVICES_DEFAULT_RESOURCES
```

Comment:
Did you add information to the DEVOPS people that you changed this ENV by adding an ```S```?
And please double-check it really goes in... the last 2 ones did not.

Reply:
Good point let me update the devops section. I will monitor the changes more closely.